tests/main/views/test_marketplace.py

```diff
@@ -551,10 +586,6 @@ def test_supplier_start_application(self):
 
     def test_supplier_applied_view_application_for_live_opportunity(self):
         self.login_as_supplier()
-        # mocking that we have applied
```

Comment:
Ah OK. Makes sense. Might be worth a comment in here as to realise the data is set up like this you need to go into a different file and spot one ID in there matches the ID of the supplier you are logged in as (which you might not remember off the top of your head)